kotlin-mui/src/main/generated/mui/material/styles/createTheme.kt

```diff
@@ -24,21 +24,7 @@ external interface ThemeOptions {
 }
 
 external interface Theme {
```

Comment:
Where is inheritance from `BaseTheme` and `SystemTheme` 
https://github.com/mui-org/material-ui/blob/master/packages/mui-material/src/styles/createTheme.d.ts#L21
https://github.com/mui-org/material-ui/blob/master/packages/mui-material/src/styles/createTheme.d.ts#L37

Reply:
Fixed in #1161